test/inductor/test_fp8.py

```diff
@@ -508,12 +511,12 @@ def __call__(self, g: torch.fx.Graph):
 
                             # Clone the inputs to potentially change stride ordering
                             a_cloned = g.call_function(
-                                torch.ops.aten.clone,
+                                torch.ops.aten.clone.default,
                                 (a_fp8,),
                                 {"memory_format": torch.contiguous_format},
                             )
                             b_cloned = g.call_function(
-                                torch.ops.aten.clone,
+                                torch.ops.aten.clone.default,
```

Comment:
This test appeared to be newly broken on this branch, but after a bit of digging I determined that it would break on `main` too by changing `clone` to `clone.default`. The new breakage occurred because `clone` is an `OpOverloadPacket`, which was previously silently skipped by `FakeTensorUpdater`. @eellison Any suggestions on how to fix this test, or would you prefer I open an issue?